ui/app/routes/jobs/job.js

```diff
@@ -35,7 +35,7 @@ export default class JobRoute extends Route {
         const relatedModelsQueries = [
           job.get('allocations'),
           job.get('evaluations'),
-          this.store.query('job', { namespace }),
+          this.store.query('job', { namespace, meta: true }),
```

Comment:
`question` `non-blocking`: We're making a second request to `/jobs/:jobId` here. Maybe we should refactor line 33 `.findRecord` to `.query` instead?